configure.sh

```diff
@@ -124,6 +130,37 @@ case $(uname -s) in
         BUILD_VIRTIO="yes"
         BUILD_MUEN="yes"
         ;;
+    OpenBSD)
+        # On OpenBSD/clang we use -nostdlibinc which gives us access to the
+        # clang-provided headers for compiler instrinsics. We copy the rest
+        # (std*.h, cdefs.h and their dependencies) from the host.
+        cc_is_clang || die "Only 'clang' is supported on OpenBSD"
+        [ "${TARGET_ARCH}" = "x86_64" ] ||
+            die "Only 'x86_64' is supported on OpenBSD"
+        if [ !ld_is_lld ]; then
```

Comment:
This is wrong, it should probably read `if ! ld_is_lld; then` -- you can't use `!` in this way with `[`.

Also, I don't understand the following warning message, did you mean "using GNU 'ld' is not supported on OpenBSD, falling back to 'ld.lld'" or something like that?